doc/man/8/mount.ceph.rst

```diff
@@ -9,7 +9,7 @@
 Synopsis
 ========
 
-| **mount.ceph** [*mon1_socket*\ ,\ *mon2_socket*\ ,...]:/[*subdir*] *dir* [
+| **mount.ceph** [*mon1_socket*\ ,\ *mon2_socket*\ ,...]:[*/subdir*] *dir* [
```

Comment:
Is the colon still required? Maybe it should be:

`[*mon1_socket*\ ,\ *mon2_socket*\ ,...][ : */subdir*] *dir*`

Reply:
I see below it looks to be required. Why not just allow omitting it? Ideally as long as `ceph.conf` is available, we could do:

`mount.ceph /mnt`

and it just works.

Reply:
No one (maybe aside from ceph developers, and not even that many of them) calls mount.ceph directly. They all call /bin/mount and that will always pass an argument for the "device" to the mount helper.

Reply:
Yeah, this will just take a benefit for developers by using "mount.ceph /mnt"

Reply:
This will also benefit the code coverage for kclient, or the relating code won't have any chance to be ran.

Reply:
That's a misuse of the mount helper, IMO. mount.ceph exists to be called from /bin/mount.

Reply:
Hmm, using /bin/mount means we must install the build every time after new changes.